### DISCUSSION TITLE
vmess不是基于tcp吗？为什么可以设置mkcp

### DISCUSSION THREAD

**Discussion by xiaoyangfeng** [v2fly/v2ray-core#1474] [General]
如题

**Comment by azzvx** [comment]
建议你多翻翻手册。
`mKCP 使用 UDP 来模拟 TCP 连接，请确定主机上的防火墙配置正确。mKCP 牺牲带宽来降低延迟。传输同样的内容，mKCP 一般比 TCP 消耗更多的流量。`https://www.v2fly.org/config/transport/mkcp.html#kcpobject
